Server/src/services/db/log.controller.ts

```diff
@@ -91,6 +91,7 @@ export default class LogController {
 	 * @param trackerId - Tracker to filter for (Optional)
 	 * @returns Log[] - List of all logs
 	 */
+	// TODO
```

Comment:
Can you maybe elaborate a bit on this TODO with a comment after it? It might be important for Darlin as well and she might not directly know what you mean by that

Reply:
Just an artefact of testing things, because I mark temporary changes like that and (normally) search and remove them before comitting. So no TODO for @Niatsuna (will be fixed in next commit).